src/System.Diagnostics.FileVersionInfo/System.Diagnostics.FileVersionInfo.sln

```diff
@@ -1,45 +1,33 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.30723.0
+# Visual Studio 14
+VisualStudioVersion = 14.0.22609.0
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "System.Diagnostics.FileVersionInfo.Tests", "tests\System.Diagnostics.FileVersionInfo.Tests.csproj", "{6DFDB760-CC88-48AE-BD81-C64844EA3CBC}"
-EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "System.Diagnostics.FileVersionInfo", "src\System.Diagnostics.FileVersionInfo.csproj", "{00EDA5FD-E802-40D3-92D5-56C27612D36D}"
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tests", "Tests", "{E89555F6-03BA-4E34-B6A7-0454CE173581}"
-EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "TestAssembly", "TestAssembly", "{A850245D-C81C-47FF-AFFA-FA4F92B7D4D9}"
-EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "System.Diagnostics.FileVersionInfo.TestAssembly", "testAssembly\System.Diagnostics.FileVersionInfo.TestAssembly.csproj", "{28EB14BE-3BC9-4543-ABA6-A932424DFBD0}"
-EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".nuget", ".nuget", "{2D0715BD-01EA-4E9D-AC88-DA5410C15183}"
 	ProjectSection(SolutionItems) = preProject
 		..\.nuget\packages.Windows_NT.config = ..\.nuget\packages.Windows_NT.config
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "System.Diagnostics.FileVersionInfo.Tests", "tests\System.Diagnostics.FileVersionInfo.Tests\System.Diagnostics.FileVersionInfo.Tests.csproj", "{6DFDB760-CC88-48AE-BD81-C64844EA3CBC}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "System.Diagnostics.FileVersionInfo.TestAssembly", "tests\System.Diagnostics.FileVersionInfo.TestAssembly\System.Diagnostics.FileVersionInfo.TestAssembly.csproj", "{28EB14BE-3BC9-4543-ABA6-A932424DFBD0}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
```

Comment:
> This isn't true. Try it: remove those elements from the .sln in notepad, then build in VS... they're not added back.

This isn't true either, sometimes it doesn't write them back, sometimes it does. This is the scenario, 

1. Delete everything under packages folder
2. Delete all configurations in notepad from .sln
3. Open sln in VS, File -> Save All
4. Open .sln in notepad, all configurations including platform specific added back.

Another scenario:
Instead of removing all configuration, just remove the platfrom specific configurations. Then when built the platform specific configurations are not added, but the Debug, and Release configurations are mapped to Windows_debug and windows_release by default. Then we don't have linux and osx configurations in the solution configuration dropdown in VS IDE. Then we can't do Go to defintions, find all references, etc for linux and osx. As in we cannot enable the unix specific files to be in compilation in VS. Which is a bad scenario for developing, as none of the VS IDE features light up for unix configurations.

In my opinion keeping these is only beneficial. What is the potential harm these can do?

>Which other solutions that have platform-specific builds have the plain Debug/Release configurations? If there are some, those should be fixed, too.

1. Microsoft.Win32.Primitives.sln,
2. InterProcessCommunication.sln,
3. System.Console.sln, 
4. System.Diagnostics.Debug.sln,
5. System.Diagnostics.Process.sln,
6. System.Diagnostics.FileVersionInfo.sln,
7. System.Diagnostics.Process.sln,
8. System.Diagnostics.TraceSource.sln,
9. System.Globalization.Extensions.sln,
10. System.IO.Compression.sln,
11. System.IO.Compression.ZipFile.sln,
12. System.IO.FileSystem.sln,
13. System.IO.FileSystem.DriveInfo.sln,
14. System.IO.FileSystem.Primitives.sln,
15. System.IO.FileSystem.Watcher.sln,
16. System.IO.MemoryMappedFiles.sln,
17. System.IO.Pipes.sln, System.Net.Http.sln,
18. System.Net.Http.WinHttpHandler.sln,
19. System.Net.Requests.sln,
20. System.Net.WebHeaderCollection.sln,
21. System.Runtime.sln,
22. System.Runtime.Extensions.sln,
23. system.Runtime.InteropServices.sln,
24. System.Runtime.InteropServices.RuntimeInformation.sln,
25. System.Security.Cryptography.*.sln,
26. System.Security.SecureString.sln,
27. System.Text.Encoding.Codepages.sln,
28. System.Threading.sln,
29. System.Xml.ReaderWriter.sln